packages/web-components/src/checkbox/checkbox.styles.ts

```diff
@@ -0,0 +1,181 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusCircular,
+  borderRadiusSmall,
+  colorCompoundBrandBackground,
+  colorCompoundBrandBackgroundHover,
+  colorCompoundBrandBackgroundPressed,
+  colorCompoundBrandForeground1,
+  colorCompoundBrandForeground1Hover,
+  colorCompoundBrandForeground1Pressed,
+  colorCompoundBrandStroke,
+  colorCompoundBrandStrokeHover,
+  colorCompoundBrandStrokePressed,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForeground3,
+  colorNeutralForegroundDisabled,
+  colorNeutralForegroundInverted,
+  colorNeutralStrokeAccessible,
+  colorNeutralStrokeAccessibleHover,
+  colorNeutralStrokeAccessiblePressed,
+  colorNeutralStrokeDisabled,
+  colorTransparentBackgroundHover,
+  fontFamilyBase,
+  fontSizeBase300,
+  lineHeightBase300,
+  spacingHorizontalS,
+  spacingHorizontalXS,
+  spacingVerticalS,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+/** Checkbox styles
+ *
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+  :host {
+    align-items: center;
+    outline: none;
+    user-select: none;
+    vertical-align: middle;
+    cursor: pointer;
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+    color: ${colorNeutralForeground3};
+  }
+  .control {
+    position: relative;
+    box-sizing: border-box;
+    display: flex;
+    flex-shrink: 0;
+    align-items: center;
+    justify-content: center;
+    overflow: hidden;
+    margin: ${spacingVerticalS} ${spacingHorizontalS};
+    fill: currentcolor;
+    height: 16px;
+    width: 16px;
+    border: ${strokeWidthThin} solid ${colorNeutralStrokeAccessible};
+    border-radius: ${borderRadiusSmall};
+    outline: none;
+  }
+  .label {
+    align-self: center;
+    cursor: inherit;
+    padding-inline: ${spacingHorizontalS} ${spacingHorizontalXS};
+    padding-bottom: ${spacingVerticalS};
+    padding-top: ${spacingVerticalS};
+  }
+  .label__hidden {
+    display: none;
+    visibility: hidden;
+  }
+  .checked-indicator {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: ${colorNeutralForegroundInverted};
+    height: 16px;
```

Comment:
I do not know what the `height` is good for but is moves the checked indicator 1px down:
<img width="241" alt="image" src="https://github.com/microsoft/fluentui/assets/9615899/91fc3651-49c6-4e1a-9612-86cbd1ef8abc">

Reply:
height was no necessary